content/applications/sales/crm/track_leads/resellers.rst

```diff
@@ -0,0 +1,130 @@
+
+Existing levels can also be edited and renamed, if desired, as well. To modify a level, select it
+from the list, and proceed to make any desired changes from the level form page that appears.
+
+Level weight is used to decide the probability a partner to be assigned a lead or opportunity. On
+the level form, assign a numerical value (greater than zero) to the :guilabel:`Level Weight` field.
+If the weight is zero, no leads are assigned.
+
+.. tip::
+   *Level Weight* can be assigned on an individual contact record. The weight assigned on the
+   individual record overwrites the default weight assigned on the level configuration form.
+
+.. _crm/partner-activations:
+
+Partner activations
+===================
+
+Partner *activations* are used to identify the status of a partner. Activations are assigned on an
+individual contact record, and can be used to group or filter the *Partnership Analysis* report
+(:menuselection:`CRM app --> Reporting --> Partnerships`).
+
+To view the partner levels, navigate to :menuselection:`CRM app --> Configuration --> Partner
+Activations`.
+
+Three activation types are created by default in the *CRM* app:
+
+- :guilabel:`Fully Operational`
+- :guilabel:`Ramp-up`
+- :guilabel:`First Contact`
+
+New partner activations can be added, as needed, by clicking :guilabel:`New`, and entering a
+:guilabel:`Name` on the new line that appears. Then, select the desired status in the
+:guilabel:`Active` column.
+
+Existing partner activations can also be edited and renamed, if desired. To rename a status, click
+the :guilabel:`Name` field of a desired level, and enter a new name.
+
+To change the active status of an activation, slide the toggle in the :guilabel:`Active` column of
+the desired activation to the *inactive* position.
+
+.. figure:: resellers/activations-toggle.png
+   :align: center
+   :alt: The list of default partner activations in the CRM app.
+
+   The list of default Partner Activations in the CRM app. The toggle for First Contact is in the
+   inactive position, while the rest are active.
+
+Partner assignments
+===================
+
+After :ref:`partner levels <crm/partner-levels>` and :ref:`partner activations
+<crm/partner-activations>` configured.
+
+To update an individual partner record, navigate to :menuselection:`CRM app --> Sales -->
+Customers`, and click the Kanban card for the desired partner to open the customer record.
+
+On the customer record, click the :guilabel:`Partner Assignment` tab.
+
+Click the :guilabel:`Partner Level` field, and select an option from the drop-down menu to assign a
+level. Click the :guilabel:`Activation` field, and select a partner activation type from the
+drop-down list, if desired. Then, click the :guilabel:`Level Weight` field to assign a different
+level weight, if necessary.
+
+Publish partners
+================
+
+With the Odoo *Website* and *Resellers* apps installed, a new webpage is created to display a list
+of all active partners from the *CRM* app.
```

Comment:
It would be helpful to show the new webpage's URL :)
```suggestion
With the Odoo *Website* and *Resellers* apps installed, a new webpage (`/partners`) is created to display a list
of all active partners from the *CRM* app.
```